vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -98,13 +98,10 @@ void executeOnPodWithSingleContainer(Map parameters, body) {
                         body()
                     } finally {
                         stashWorkspace(config, 'container')
-                     }
+                    }
                 }
             }
         }
-    } catch (e) {
-        stashWorkspace(config, 'container')
-        throw e
     } finally {
         unstashWorkspace(config, 'container')
```

Comment:
This unstash statement will fail if there is an exception in the pod creation.

Reply:
It will, but exception is reported and ignored.
The workspace will stay the same.

Reply:
Thats right. Missed the exception handling in `unstashWorkspace`.